lib/jasmine-reportportal-reporter.js

```diff
@@ -52,23 +56,57 @@ class ReportportalReporter {
     }
 
     setDescription(description) {
-        this.additionalCustomParams = Object.assign(this.additionalCustomParams, description && { description: description.text });
+        const { suiteDescription } = this.suiteAdditionalCustomParams;
+
+        if (description && description.text.suite) {
+            this.suiteAdditionalCustomParams.suiteDescription = suiteDescription.concat(description.text);
+        } else {
+            this.additionalCustomParams = Object.assign(this.additionalCustomParams, description && { description: description.text });
+        }
     }
 
     registerListeners () {
         process.on(EVENTS.ADD_ATTRIBUTES, this.addAttributes.bind(this));
         process.on(EVENTS.ADD_DESCRIPTION, this.setDescription.bind(this));
     }
 
+    getSuiteAttributesBySuite (suite) {
+        const { suiteAttributes } = this.suiteAdditionalCustomParams;
+
+        return suiteAttributes && suiteAttributes.filter(attr => attr.suite === suite).map(attr => ({
+            key: attr.key,
+            value: attr.value,
+        }));
+    }
+
+    getSuiteDescriptionBySuite (suite) {
+        const { suiteDescription } = this.suiteAdditionalCustomParams;
+
+        return suiteDescription && suiteDescription.filter(text => text.suite === suite).map(item => item.text);
```

Comment:
It looks easier to use a Map with suite description as a key to store suite descriptions.